```
FAILED tests/app/test_agents.py::TestAgentUnitMocked::test_configured_retries_wired_into_agent - AttributeError: 'Agent' object has no attribute '_max_output_retries'. Did you mean: '_max_result_retries'?
```

looks related

--- a/packages/app/setup.cfg
+++ b/packages/app/setup.cfg
@@ -49,7 +49,7 @@ install_requires =
     pebble
     pulsar-galaxy-lib>=0.15.0.dev0
     pydantic>=2.7.4
-    pydantic-ai>=0.1.16
+    pydantic-ai>=1.99.0
     pysam>=0.21
     python-slugify
     PyJWT

